runelite-client/src/main/java/net/runelite/client/plugins/regenmeter/RegenMeterConfig.java

```diff
@@ -57,4 +57,13 @@ default boolean showWhenNoChange()
 	{
 		return false;
 	}
+
+	@ConfigItem(
+		keyName = "regenWarning",
+		name = "Show a warning when about to regenerate health.",
```

Comment:
You should probably follow the naming for notification alers in Idle Notifier (e.g for HP etc)